### PR TITLE
hometask 4

### PR DESCRIPTION
**Pull request by MaxSkydan**

### CODE REVIEW

File: count_vowels.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+"""
+Hometask 4 - Count vowels.
+
+# The English language has five vowels: A, E, I, O, and U
```
Comment (antohaUa) [inline count_vowels.py]: Comment signs could be removed - just raw text of task description

File: count_vowels.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+for vowel, count in vowels.items():
+    print(f'| {vowel:^6}|  {count:^8} |')
+print('-' * sep_num)
+
+"""
+We modify the text in which each vowel is replaced.
+And after that we display this changed text.
+"""
+
+modify = str.maketrans('AaEeIiOoUu', 'ÀàÉéÍíÓóÚú')
```
Comment (antohaUa) [inline count_vowels.py:59]: Yes! This is the best option here for replace.

File: count_vowels.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Along the margin of a bay:
+Ten thousand saw I at a glance,
+Tossing their heads in sprightly dance."""
+
+"""
+We count the vowels in the text and put them in the dictionary.
+And after that we display a table with the counted vowels.
+"""
+
+vowels = {'A': poem_text.count('a'), 'E': poem_text.count('e'),
```
Comment (antohaUa) [inline count_vowels.py]: Pay attention that you need to get count of  lower and capital letters ( count 'A' + count 'a') or do it somehow in more smart way. Now solution gives wrong result/ It should be fixed

**Review by antohaUa — CHANGES_REQUESTED** [review]
Bug detected )

File: count_vowels.py (hunk 1 above)
Comment (MaxSkydan) [reply count_vowels.py]: I commented out the table example because without it the linter gives errors

**Comment by MaxSkydan** [comment]
Fixed bug

File: count_vowels.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+Fluttering and dancing in the breeze.
+Continuous as the stars that shine
+And twinkle on the Milky Way,
+They stretched in never-ending line
+Along the margin of a bay:
+Ten thousand saw I at a glance,
+Tossing their heads in sprightly dance.
+"""
+
+"""
```
Comment (antohaUa) [inline count_vowels.py]: Better to use comment lines starting with #
These lines you can spread within you code ->  separate each logical block with certain comment.

Or if you describe whole algorithm then these lines you need to put into docstring - right after task description.

**Review by antohaUa — APPROVED** [review]
Good

**Pull request merged**